src/app/modules/candidate-registration/components/result/result.component.ts

```diff
@@ -36,8 +36,6 @@ export class ResultComponent implements OnInit {
       .pipe(finalize(() => this.loadingService.hide()))
       .subscribe((response) => {
         this.candidateDetails = JSON.parse(response.body);
-
-        console.log('this.candidateDetails', this.candidateDetails);
```

Comment:
keep it as it is, Already covered under https://github.com/teamclairvoyant/clairthon-rising-hackatoons/pull/34 , Just to avoid merge conflicts